database/Seeders/eggs/source-engine/egg-ark--survival-evolved.json

```diff
@@ -100,22 +92,31 @@
             "rules": "required|numeric"
         },
         {
-            "name": "Auto-update server",
```

Comment:
Why remove this option when it exists and is useful, because breaking the whole server with outdated mods is not ideal